javascript/javascript3/week3/promis.js

```diff
@@ -0,0 +1,40 @@
+async function getUsers(userName) {
+  let url = ` https://api.github.com/search/repositories?q=user:${userName}`;
+  try {
+    let res = await fetch(url);
+    return await res.json();
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+console.log(getUsers("benna100"));
+const userDisplay = document.getElementById("container");
+const promiseAll = Promise.all([
+  getUsers("seenachacko"),
+  getUsers("Gohar33"),
+  getUsers("slaviaF"),
+]);
+promiseAll.then((data) => console.log(data));
+promiseAll.then((data) => {
+  data.forEach((user) => {
+    console.log(user.items[0].owner.login);
```

Comment:
The console logging logic could be put into a function that does just that. This would make it easier to read and maintain.